dev_guide/deployments.adoc

```diff
@@ -273,6 +273,14 @@ image stream changes and the new tag value differs from the current image
 specified in the deployment configuration's `helloworld` container, a new
 deployment is created using the new tag value for the `helloworld` container.
 
+The following table explains the possible combinations between ConfigChange
+and ImageChange triggers.
+
+  Trigger                  | true                                      | false
+  -----------------------  | ----------------------------------------- | ------------------------------------------------------
+  ConfigChange             | Deployments are started automatically     | Deployments need to be started manually
+  ImageChange (automatic)  | Images are always resolved automatically  | Only the initial image will be resolved in the template
```

Comment:
I think this can work but i was thinking of a table like:
```                       
            configchange | true | false
-----------------------------------------
ICT |
-----------------------------------------
true |                            X   |      Y
------------------------------------------
false |                          Z   |       W
```

that's harder to extend if we add more behavior later though.

at a minimum i'd change "images are always resolved automatically" to "Image value is resolved from the imagestreamtag referenced in the imagechangetrigger".  That may not fix in a box though :)

the important thing someone needs to be able to take away from this table is understanding the interaction of the settings...  i'm not sure knowing that "images are resolved automatically" and "deployments are started automatically" would lead me to realize that if I have an ICT=true and a CCT, that i'm going to get deployments when the image changes (because i don't realize that "images are resolved automatically" really means "the pod template is updated automatically which will result in the CCT firing"